docker/docker/src/main/java/org/arquillian/cube/docker/impl/client/metadata/ReportMetrics.java

```diff
@@ -0,0 +1,102 @@
+package org.arquillian.cube.docker.impl.client.metadata;
+
+import org.arquillian.cube.docker.impl.client.config.CubeContainer;
+import org.arquillian.cube.docker.impl.model.DockerCube;
+import org.arquillian.cube.spi.metadata.CanReportMetrics;
+import org.arquillian.recorder.reporter.Reportable;
+import org.arquillian.recorder.reporter.model.entry.GroupEntry;
+import org.arquillian.recorder.reporter.model.entry.KeyValueEntry;
+import org.arquillian.recorder.reporter.model.entry.table.TableCellEntry;
+import org.arquillian.recorder.reporter.model.entry.table.TableRowEntry;
+
+import java.security.Key;
+import java.util.Collection;
+
+/**
+ * Reporting metrics capabilities for Docker Cube.
+ */
+public class ReportMetrics implements CanReportMetrics {
+
+    private DockerCube dockerCube;
+
+    public ReportMetrics(DockerCube dockerCube) {
+        this.dockerCube = dockerCube;
+    }
+
+    @Override
+    public Reportable report() {
+
+        GroupEntry groupEntry = new GroupEntry(dockerCube.getId());
+
+        boolean error;
+        switch(dockerCube.state()) {
```

Comment:
If that's an enum we can use `EnumSet.of(....).contains(dockerCube.state())` instead of switch.